website/docs/docs/cloud/migration.md

```diff
@@ -11,15 +11,15 @@ dbt Labs is in the process of rolling out a new cell-based architecture for dbt
 
 We're scheduling migrations by account. When we're ready to migrate your account, you will receive a banner or email communication with your migration date. If you have not received this communication, then you don't need to take action at this time. dbt Labs will share information about your migration with you, with appropriate advance notice, when applicable to your account.
 
-Your account will be automatically migrated on its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
+Your account will be automatically migrated on or after its scheduled date. However, if you use certain features, you must take action before that date to avoid service disruptions.
 
 ## Recommended actions
 
 We highly recommended you take these actions:
 
-- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations will be voided during the migration and must be resent after it is complete.
-- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes will be lost during migration.
-- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they will be lost during migration. If you lose critical logs older than 90 days during the migration, you will have to work with the dbt Labs Customer Support team to recover.
+- Ensure pending user invitations are accepted or note outstanding invitations. Pending user invitations may be voided during the migration. You can resend user invitations after the migration is complete.
+- Commit unsaved changes in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).  Unsaved changes may be lost during migration.
+- Export and download [audit logs](/docs/cloud/manage-access/audit-log) older than 90 days, as they may be unavailable from dbt Cloud after the migration is complete. Logs older than 90 days while within the data retention period are not deleted, but you will have to work with the dbt Labs Customer Support team to recover.
```

Comment:
"Unavailable" reads like they could be made available again. We should stick with language that firmly states they are gone. We don't have to use lost, maybe deleted, purged, or something similar. Something direct.

Reply:
I think the best description is "will be unavailable from dbt Cloud". I will update this.